src/btree/bt_split.c

```diff
@@ -1038,8 +1036,10 @@ __split_internal(WT_SESSION_IMPL *session, WT_PAGE *parent, WT_PAGE *page)
     __wt_timing_stress(session, WT_TIMING_STRESS_SPLIT_5);
 
     /* Split into the parent. */
-    WT_ERR(__split_parent(
-      session, page_ref, alloc_index->index, alloc_index->entries, parent_incr, false, false));
+    if ((ret = __split_parent(session, page_ref, alloc_index->index, alloc_index->entries,
```

Comment:
`WT_ERR` will `goto error` if the call returns non-zero, so there's no need to make any of these into an `if` clause - you can simply put the stat increment after the `WT_ERR` and it will only run in the success case.

Reply:
@korteland 
I have change the code, please review again.
thanks.